src/main/java/dataStructures/avlTree/AVLTree.java

```diff
@@ -372,6 +372,10 @@ private T successor(Node<T> node) {
         return null;
     }
 
+
```

Comment:
Should we add this to all the current implementations (eg. segment portions of code based on it's purpose) for consistency?